stable/traefik/templates/_helpers.tpl

```diff
@@ -7,3 +7,16 @@ We truncate at 63 chars because some Kubernetes name fields are limited to this
 {{- define "fullname" -}}
 {{- printf "%s-%s" .Release.Name .Chart.Name | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
+
+{{/*
+Create the block for the ProxyProtocol's Trusted IPs.
+This simplifies some duplicate code.
```

Comment:
> This simplifies some duplicate code.

This is the well-understood function of a helper. It doesn't have to be stated. Suggest removing that comment.